src/variables/constants.ts

```diff
@@ -22,6 +24,18 @@ export const environmentTypes: IEnvironmentTypes = {
     local: 'local'
 };
 
+export const epsUrls: IEpsUrls = {
+    production: '',
+    staging: '',
+    local: ''
+};
+
+export const epsBucketUrls: IEpsBucketUrls = {
+    production: '',
+    staging: '',
+    local: ''
+};
+
```

Comment:
If the Url is already provided in the IEnvironment why is this required? And if required, why `checkout-frontend-library` is the place for it?